### ISSUE TITLE
replace react-easy-panzoom in dm area with our own implementation

### ISSUE THREAD

**Issue by n1ru4l** [dungeon-revealer/dungeon-revealer#461]
Some time ago I replaced the player-area pan zoom view with a self-crafted one that did not cause any issues on mobile safari. It performs better in general and we should also use it in the dm area. We might have to add some missing features to our pan-zoom view before that.

**Comment by n1ru4l** [comment]
I now have an iPad and the player area is basically unusable. It crashes often upon loading (even with small maps). If it does not crash upon losing it does after doing minimal interaction (not even triggering a touch gesture).

I am using an iPad 2019 10.2".

@GoOz You mentioned not having any issues with maps of any size. Could you give more information about your environment? (https://github.com/dungeon-revealer/dungeon-revealer/issues/450#issuecomment-631377399)

Given the current status, I guess the only viable solution for having good performance on mobile devices/tablets is going full canvas. I already did some testing with [react-three-fiber](https://github.com/react-spring/react-three-fiber) and it seems promising.

It would probably also make https://github.com/dungeon-revealer/dungeon-revealer/issues/433 easier to implement (in a performant way), as it is possible to use video textures with there's (https://threejs.org/docs/#api/en/textures/VideoTexture).

**Comment by GoOz** [comment]
@n1ru4l Mine is 6th gen, 9,7″ screen, iOS 13.4.1

I indeed had no issue using DR on the player view with the basic tools such as scrolling & zooming **but** I noticed recently it had some sync issues from time to time, but I couldn't find any reproducible steps.

What I noticed is that sometimes it loses sync, like when I save some cleared fog of war on the DM view, it doesn't clear on the player view and the location ping tool doesn't work either. Sometimes just one of the two, sometimes both at the same time.

But usually a simple refresh for both DM & player views fixes this behavior.

Is that what you call a crash? Because I never had a real crash of the app, like the app is still running well on docker or locally.
